src/ScriptVerification/Lib.agda

```diff
@@ -34,7 +34,10 @@ createEnv s = record { slot = s ; treasury = 0 ;
                                ; minUTxOValue = 0
                                ; poolDeposit = 500000000 -- lovelace
                                ; keyDeposit = 500000000 -- lovelace
-                               ; coinsPerUTxOByte = 4310 --lovelace
+                               ; coinsPerUTxOByte = 0  --lovelace
+                               -- ^^^ was 4310, but that value resulted in failed tests once
```

Comment:
Sounds like tests need to be fixed. @williamdemeo, could you please create a ticket about it so we don't forget to fix 'em.